src/Entity/OgMembershipType.php

```diff
@@ -52,4 +69,40 @@ public function id() {
     return $this->type;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function save() {
+    $status = parent::save();
+
+    if ($this->isSyncing()) {
+      // Do not create config while config import is in progress.
+      return;
+    }
+
+    if ($status === SAVED_NEW) {
+      FieldConfig::create([
+        'field_name' => 'og_membership_request',
+        'entity_type' => 'og_membership',
+        'bundle' => $this->id(),
+        'label' => 'Request Membership',
+        'description' => 'Explain the motivation for your request to join this group.',
+        'translatable' => TRUE,
+        'settings' => [],
+      ])->save();
+    }
+
+    return $status;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function delete() {
+    if ($this->id() === OgMembershipInterface::TYPE_DEFAULT) {
```

Comment:
This has two problems:
1. The "throw" line should be "throw new \Exception".
2. It makes it impossible to install OG.